### ISSUE TITLE
Exporting radius_graph as torchscript fails: python value of type 'module' cannot be used as a value.

### ISSUE THREAD

**Issue by janblumenkamp** [pyg-team/pytorch_geometric#4046]
### 🐛 Describe the bug

I am trying to export a `nn.Module` that uses the pooling layer `radius_graph` as torchscript. This fails as described below.

```
import torch
import torch.nn as nn
from torch_geometric.nn import radius_graph

class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()

    def forward(self, x, batch):
        return radius_graph(x, r=1.5, batch=batch, loop=False)

x = torch.Tensor([[-1, -1], [-1, 1], [1, -1], [1, 1]])
batch = torch.tensor([0, 0, 0, 0])

model = Model()
scripted = torch.jit.script(model, (x, batch))
print(scripted)
```

Traceback:
```
Traceback (most recent call last):
  File "/home/jan/Documents/PhD/ros2_tb_ctrl/test_torchscript.py", line 16, in <module>
    scripted = torch.jit.script(model, (x, batch))
  File "[...]/python3.9/site-packages/torch/jit/_script.py", line 1096, in script
    return torch.jit._recursive.create_script_module(
  File "[...]/python3.9/site-packages/torch/jit/_recursive.py", line 412, in create_script_module
    return create_script_module_impl(nn_module, concrete_type, stubs_fn)
  File "[...]/python3.9/site-packages/torch/jit/_recursive.py", line 478, in create_script_module_impl
    create_methods_and_properties_from_stubs(concrete_type, method_stubs, property_stubs)
  File "[...]/python3.9/site-p...]/ackages/torch/jit/_recursive.py", line 355, in create_methods_and_properties_from_stubs
    concrete_type._create_methods_and_properties(property_defs, property_rcbs, method_defs, method_rcbs, method_defaults)
  File "[...]/python3.9/site-packages/torch/jit/_recursive.py", line 795, in try_compile_fn
    return torch.jit.script(fn, _rcb=rcb)
  File "[...]/python3.9/site-packages/torch/jit/_script.py", line 1143, in script
    fn = torch._C._jit_script_compile(
RuntimeError: 
python value of type 'module' cannot be used as a value. Perhaps it is a closed over global variable? If so, please consider passing it in as an argument or use a local varible instead.:
  File "[...]/python3.9/site-packages/torch_geometric/nn/pool/__init__.py", line 210
        edge_index = radius_graph(x, r=1.5, batch=batch, loop=False)
    """
    if torch_cluster is None:
       ~~~~~~~~~~~~~ <--- HERE
        raise ImportError('`radius_graph` requires `torch-cluster`.')
'radius_graph' is being compiled since it was called from 'Model.forward'
  File "/home/jan/Documents/PhD/ros2_tb_ctrl/test_torchscript.py", line 10
    def forward(self, x, batch):
        return radius_graph(x, r=1.5, batch=batch, loop=False)
               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <--- HERE
```

### Environment

* PyG version: 1.7.2
* PyTorch version: 1.9.0
* OS: Manjaro Linux
* Python version: 3.9.9
* CUDA/cuDNN version: 10.2
* How you installed PyTorch and PyG (`conda`, `pip`, source): pipi/system
* Any other relevant information (*e.g.*, version of `torch-scatter`):

**Comment by rusty1s** [comment]
Thanks for reporting. I think I fixed this in #4047. As a simple workaround, you can just import `radius_graph` from `torch_cluster` rather than `torch_geometric.nn` and everything should work as expected:
```python
import torch
import torch.nn as nn
from torch_cluster import radius_graph

class Model(nn.Module):
    def __init__(self):
        super(Model, self).__init__()

    def forward(self, x, batch):
        return radius_graph(x, r=1.5, batch=batch, loop=False)

x = torch.Tensor([[-1, -1], [-1, 1], [1, -1], [1, 1]])
batch = torch.tensor([0, 0, 0, 0])

model = Model()
scripted = torch.jit.script(model)
print(scripted)
```

**Comment by janblumenkamp** [comment]
Awesome, thanks so much for the quick and helpful reply! Your workaround works!
